scss/_sjc-header.scss

```diff
@@ -2,8 +2,8 @@
 
 .omnibar .navbar {

```

Comment:
Let's change all of these to be nested within one `.omnibar` selector for the whole file:

```scss
.omnibar {
  .navbar-brand {}
  .navbar-brand a {}
  .global-icons .nav-link {}
  // etc
}
```